packages/clerk-js/package.json

```diff
@@ -66,21 +66,17 @@
     "@babel/preset-typescript": "^7.12.1",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.7",
     "@svgr/webpack": "^6.2.1",
-    "@testing-library/dom": "^7.28.1",
-    "@testing-library/jest-dom": "^5.11.6",
-    "@testing-library/react": "^11.2.1",
-    "@testing-library/react-hooks": "^3.4.2",
-    "@testing-library/user-event": "^12.2.2",
+    "@testing-library/dom": "^8.19.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^14.4.3",
     "@types/classnames": "^2",
     "@types/jest": "^27.4.0",
     "@types/jsonwebtoken": "^8",
     "@types/qs": "^6.9.3",
-    "@types/react": "^17.0.39",
-    "@types/react-dom": "^17.0.11",
-    "@types/react-test-renderer": "^17",
+    "@types/react": "*",
+    "@types/react-dom": "*",
     "@types/sass": "^1",
-    "@types/testing-library__jest-dom": "^5",
-    "@types/testing-library__react-hooks": "^3",
```

Comment:
❓ Are these deprecated?

Reply:
Not deprecated per se, however the latest testing-library version has been restructured to include re-export these lower-level libs and types, so I'm fairly confident we no longer need these.

In general, global `@types/*` can cause issues (conflicting type declarations or duplicate declarations) in monorepos when packages are using different versions or if they depend on packages that use conflicting `@type/`. For instance, we've fought build issues in the past caused by conflicts between different versions of node and dom types introduced by dependencies in the global space.

In general, the less global `@types` we use, the better